Source/WebInspectorUI/UserInterface/Views/ObjectTreePropertyTreeElement.css

```diff
@@ -84,7 +84,7 @@
     font-size: 12px;
 }
 
-.object-tree-property .property-name.not-enumerable {
+.object-tree-property .property-name:is(.private, .not-enumerable) {
     opacity: 0.6;
```

Comment:
Aside: it would be nice if the text were just one of our standard colors (like secondary) instead of dimming text with opacity.

Reply:
yeah id actually like to do a followup to change this to be `.internal` instead of `.not-enumerable`.  i dont think most developers realize that semi-transparent means "not enumerable" 😅